cmd/bridge/main.go

```diff
@@ -660,17 +650,25 @@ func main() {
 				http.Redirect(res, req, redirectURL.String(), http.StatusMovedPermanently)
 			})
 			redirectPort := fmt.Sprintf(":%d", *fRedirectPort)
-			log.Infof("Listening on %q for custom hostname redirect...", redirectPort)
-			log.Fatal(http.ListenAndServe(redirectPort, redirectServer))
+			klog.Infof("Listening on %q for custom hostname redirect...", redirectPort)
+			klog.Fatal(http.ListenAndServe(redirectPort, redirectServer))
 		}()
 	}
 
-	log.Infof("Binding to %s...", httpsrv.Addr)
+	klog.Infof("Binding to %s...", httpsrv.Addr)
 	if listenURL.Scheme == "https" {
-		log.Info("using TLS")
-		log.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
+		klog.Info("using TLS")
+		klog.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
 	} else {
-		log.Info("not using TLS")
-		log.Fatal(httpsrv.ListenAndServe())
+		klog.Info("not using TLS")
+		klog.Fatal(httpsrv.ListenAndServe())
+	}
+}
+
+func setLogLevel(val string) (string, error) {
```

Comment:
@zherman0 don't think this function is needed, since it's not used anywhere. Do you recall whats the purpose by any chance, so we dont overlook anything.

Reply:
This function is only for anywhere that people might still be using the the log level so we can informed them it is deprecated.  It is not needed but I don't think it hurts to tell people of the change.  If we leave it, then we should probably remove it in a release or 2.